docs/api.md

```diff
@@ -92,6 +92,7 @@ _Appears in:_
 | Field | Description | Default | Validation |
 | --- | --- | --- | --- |
 | `simplified` _[SimplifiedStatus](#simplifiedstatus)_ |  |  |  |
+| `inFlightSynthesis` _[Synthesis](#synthesis)_ |  |  |  |
```

Comment:
Maybe we should start adding more comments to the api structs so we can use this auto-gen files as proper docs

Reply:
Agreed - we're pretty close on a few structs but most of them are still lacking